mesonbuild/modules/gnome.py

```diff
@@ -884,12 +884,16 @@ def gdbus_codegen(self, state, args, kwargs):
         xml_files = args[1:]
         target_name = namebase + '-gdbus'
         cmd = [self.interpreter.find_program_impl('gdbus-codegen')]
+        extra_args = mesonlib.stringlistify(kwargs.pop('extra_args', []))
+        scan_command += extra_args
```

Comment:
__[Flake8]__

[F841] local variable 'scan_command' is assigned to but never used


<!-- Comment by [SideCI](https://sideci.com) -->